mlir/test/fusion/tosa-to-miopen-relu.xmir.mlir

```diff
@@ -1,4 +1,5 @@
 // RUN: mlir-miopen-driver -host-pipeline full -kernel-pipeline full -triple amdgcn-amd-amdhsa -target gfx908 %s | FileCheck %s
+// XFAIL: *
```

Comment:
FYI: Will address as a independent topic.

Upstream (https://reviews.llvm.org/D119444) break us by removing essential lowering patterns. Needs to find out the equivalent available ones.

Reply:
https://github.com/ROCmSoftwarePlatform/llvm-project-private/issues/521